lightningd/channel/channel.c

```diff
@@ -271,6 +263,9 @@ static void init_channel(struct peer *peer, const u8 *msg)
 				    &points[LOCAL], &points[REMOTE],
 				    am_funder ? LOCAL : REMOTE);
 
+	peer->channel_direction =
+	    pubkey_cmp(&peer->node_ids[LOCAL], &peer->node_ids[REMOTE]) > 0;
+
```

Comment:
Disappointing we can't use it here too.  We should at least put a fixme in this case, for revisiting later (or put it inline in routing.h).